docs/docs/basil.md

```diff
@@ -236,22 +252,22 @@ The first thing we need to do, is tell our app to link to the `openzeppelin-zos`
 
 ```sh
 zos link openzeppelin-zos
-zos push --from $OWNER --deploy-stdlib --network local
+zos push --from $OWNER --deploy-stdlib --network development
 ```
 
-Notice the `--deploy-stdlib` option we've used. What this does is inject a version of the standard lib in our development network. Since we're working on a local blockchain, ZeppelinOS's contracts don't exist. This handy option solves that problem for us quite conveniently ^^
+Notice the `--deploy-stdlib` option we've used. What this does is inject a version of the standard lib into our development network. Since we're working on a local blockchain, the smart contracts that make up the ZeppelinOS library don't exist there. This handy option solves that problem for us quite nicely.
 
```

Comment:
I like most changes here but why "conveniently" -> "nicely"?

Also, the "^^" is supposed to be an emoticon :)